caddy/caddy.go

```diff
@@ -55,6 +60,8 @@ type workerConfig struct {
 	Env map[string]string `json:"env,omitempty"`
 	// Directories to watch for file changes
 	Watch []string `json:"watch,omitempty"`
+	// ModuleID identifies which module created this worker
+	ModuleID string `json:"module_id,omitempty"`
```

Comment:
Why is this a string? Because caddy can randomly decide to shift modules around in memory, so using the address does not work reliably! It also doesn't expose any of the information from before the php_server directive, so we're left with this.

Hate it.